src/ert/com_protocol/_schema.proto

```diff
@@ -54,19 +84,20 @@ message JobId {
 
 message Experiment {
   required ExperimentId id   = 1; // NOTE: in the message the id is optional since it often will be streamed
-  optional Status status   = 2 [default = UNKNOWN];
+  optional ExperimentStatus status   = 2 [default = EXPERIMENT_STARTED];
+  optional string message = 3;
 
   map<string, Ensemble> ensembles= 101;
 }
 message Ensemble {
   required EnsembleId id   = 1;
-  optional Status status   = 2 [default = UNKNOWN];
+  optional EnsembleStatus status   = 2 [default = ENSEMBLE_STARTED];
```

Comment:
Should we have another possible state for the default, something to say that the ensemble has been constructed or something, but not yet sent off for evaluation (but maybe that is an impossible state if the state machine does not exist prior to evaluation, don't know the code good enough..)